tests/dummy/app/templates/docs/validation.md

```diff
@@ -0,0 +1,23 @@
+# Validation
+
+Built in components to validate Caluma data.
+
+## Usage
+
+### DocumentValidity
+
+By passing a document to this component checks the validity of all its questions.
```

Comment:
It's pretty relevant that it's only validated if this component enters the viewport. Add this and then the docs are alright

Reply:
Added this line